gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Sep 13 18:15:04 IST 2019
+#Mon Nov 18 10:47:27 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
@nikitamarysolomanpvt Are these changes necessary?

Reply:
Iam facing issue while using this update from develop of gradle version gradle-5.1.1 . We could raise an issue and fix this if anybody facing issue after upgrading to 5.4.1.

Reply:
Okay, for now we should merge this.

Reply:
@nikitamarysolomanpvt can you include the issue you're facing here for posterity? It's helpful if someone comes back to this comment in the future & wonders why we made the gradle version change.

Reply:
Now i'am not facing this issue. If it occurs again will mention. Was using 5.4.1. only before don't know why it's changed.

Reply:
<img width="1577" alt="small" src="https://user-images.githubusercontent.com/54615666/69135475-28665000-0adf-11ea-95cd-b54840bd7f54.png">